src/Compilers/CSharp/Test/Semantic/FlowAnalysis/FlowDiagnosticTests.cs

```diff
@@ -1207,15 +1207,37 @@ struct S
     int x, y;
     S(int x) { this.x = x; }
 }";
-            CreateCompilation(program)
+            CreateCompilation(program, parseOptions: TestOptions.Regular10)
                 .VerifyDiagnostics(
                 // (5,5): error CS0171: Field 'S.y' must be fully assigned before control is returned to the caller
                 //     S(int x) { this.x = x; }
-                Diagnostic(ErrorCode.ERR_UnassignedThis, "S").WithArguments("S.y"),
+                Diagnostic(ErrorCode.ERR_UnassignedThis, "S").WithArguments("S.y").WithLocation(5, 5),
+                // (5,5): error CS8652: The feature 'auto default struct fields' is currently in Preview and *unsupported*. To use Preview features, use the 'preview' language version.
```

Comment:
Could we add the name of the field to the LangVer diagnostic? Ideally we wouldn't have two errors. Just one would suffice (You're using unassigned field, if you want to use auto-default then use LangVer preview).

Reply:
I've thought a bit about it. I feel satisfied with the current quality of the diagnostics and uncertain whether the additional effort to combine the diagnostics is warranted. Let me know if you feel differently here.